velox/exec/fuzzer/MemoryArbitrationFuzzer.cpp

```diff
@@ -94,6 +94,12 @@ DEFINE_int32(
     "filesystem. This is only applicable when 'spill_faulty_fs_ratio' is "
     "larger than 0");
 
+DEFINE_int32(
+    seconds_to_abort,
```

Comment:
```
s/seconds_to_abort/task_abort_interval_ms/
```
Let's define in millisecond granularity which is more flexible for test control.